lib/linguist/heuristics.rb

```diff
@@ -89,6 +97,18 @@ def initialize(exts_and_langs, rules)
       @rules = rules
     end
 
+    # Internal: Return the heuristic's target extensions
+    def extensions
+      @exts_and_langs
```

Comment:
Not introduced by this PR, but shouldn't `exts_and_langs` be renamed to `exts` or `extensions`?

Reply:
It should, and I found that name particularly misleading myself. I'll change it.